chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -228,6 +237,8 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
 
             it("then the correct actions will be dispatched") {
                 verify(exactly = 1) {
+                    dispatcher(CurrentUserReceived(userApiTypeMapper.toUserInternalType(simpleUser)))
+                    dispatcher(CurrentUserReceived(currentUser = userApiTypeMapper.toUserInternalType(simpleUser)))
```

Comment:
Minor style: This is pushing the line length, perhaps break it to match the one below?

Reply:
👍 this is also had a bug where 240 and 241 did the same thing and shouldn't be written twice. Have deleted 240 and spread 241 over multiple lines!